sdk/dataprotection/Azure.ResourceManager.DataProtectionBackup/src/Customized/Models/DataProtectionOperationJobExtendedInfo.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+using Azure.Core;
+
+namespace Azure.ResourceManager.DataProtectionBackup.Models
+{
+    public partial class DataProtectionOperationJobExtendedInfo
+    {
+        /// <summary> Arm Id of the job created for this operation. </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Guid? JobId { get; }
```

Comment:
Since this will always return `null`, and there is no chance we can convert `JobResourceId` to `JobId` (we cannot convert `ResourceId` to `UUID`), maybe we just delete this property?
@ArthurMa1978 What's your opinion?

Reply:
There will be API compact issue if we remove this property.

Reply:
Yes, we need to keep the old one